launcher/ui/pages/instance/InstanceSettingsPage.cpp

```diff
@@ -437,6 +454,70 @@ void InstanceSettingsPage::on_javaTestBtn_clicked()
     checker->run();
 }
 
+void InstanceSettingsPage::updateAccountsMenu()
+{
+    accountMenu->clear();
+
+    auto accounts = APPLICATION->accounts();
+    int accountIndex = accounts->findAccountByProfileId(m_settings->get("InstanceAccountId").toString());
+
+    if (accountIndex != -1)
+    {
+        auto account = accounts->at(accountIndex);
+        ui->instanceAccountSelector->setText(account->profileName());
+        ui->instanceAccountSelector->setIcon(account->getFace());
+    } else {
+        ui->instanceAccountSelector->setText(tr("No default account"));
+        ui->instanceAccountSelector->setIcon(APPLICATION->getThemedIcon("noaccount"));
```

Comment:
I'm not sure what this means. Even with the instance-specific override, leaving this selected doesn't prompt the user to select an account at launch (as one would expect from a 'No default account' option). It simply uses the more generic launcher-wide default account. Maybe this should be rephrased?

Reply:
As @TheKodeToad pointed out, the text for the checkbox was a little verbose. I reworded the checkbox to "Override default account", and when unchecked, the greyed-out account dropdown will mirror the global dropdown in the main launcher window (ie, it will show the currently selected default account or show "No default account" if none is selected)